packages/mds-repository/package.json

```diff
@@ -24,12 +24,12 @@
     "@mds-core/mds-types": "workspace:0.9.2",
     "@mds-core/mds-utils": "workspace:0.5.2",
     "@types/pg": "8.6.5",
-    "await-lock": "2.1.0",
-    "envalid": "7.2.1"
+    "await-lock": "2.2.2",
+    "envalid": "7.3.1"
   },
   "devDependencies": {
     "pg": "8.7.3",
-    "typeorm": "0.3.4"
+    "typeorm": "0.3.6"
```

Comment:
The `typeorm` version should also be updated in `peerDependencies` below.

Reply:
Fixed in [9e91e3f](https://github.com/lacuna-tech/mds-core/pull/1003/commits/9e91e3f5bff1a8c22bd27f197cd1ba9263b8433a), thanks!